bd-crash-handler/src/lib.rs

```diff
@@ -291,21 +291,20 @@ impl Monitor {
         // TODO(snowp): Eventually we'll want to upload the report out of band, but for now just
         // chuck it into a log line.
         logs.push(CrashLog {
-          fields: vec![
-            AnnotatedLogField::new_ootb("_crash_artifact".into(), contents.into()).into(),
-            AnnotatedLogField::new_ootb(
+          fields: [
+            ("_crash_artifact".into(), contents.into()),
```

Comment:
Another follow up that I think we should do at this point is that there are many places where the key is just a static string. I think based on how you have it setup you could pretty easily have a FieldKey type which operates on &'static string. You could have some of these be lazy init Arc but that seems more error prone vs. just supporting it directly. There is no reason to copy these things over and over, especially for the higher rate ones.

Reply:
I just switched FIeldKey to be a Cow which was relatively painless, seems like a nice improvement 

We could potentially also support Arc<String> but we wouldn't benefit from that in many places so maybe not worth the complexity. Using a symbol store might be helpful to avoid re-creating the same keys over and over again but that's a bigger project